app/uk/gov/hmrc/gform/auditing/AuditService.scala

```diff
@@ -43,13 +45,18 @@ trait AuditService {
       case _ => false
     }))
 
-    val processedData: Seq[FormField] = {
+    val processedData: Seq[FormField] = if (optSortCode.nonEmpty) {
       optSortCode.flatMap { fieldValue =>
         UkSortCode.fields(fieldValue.id).flatMap { fieldId =>
           val sortCode: String = form.formData.fields.filter(_.id == fieldId).map(_.value).mkString("-")
-          form.formData.fields.filterNot(_.id == fieldId) ++ Seq(FormField(fieldValue.id, sortCode))
+          val xc = form.formData.fields.filterNot(_.id == fieldId)
+          Logger.debug("SOmething here " + xc)
```

Comment:
Looks like this logging statement is dev-phase legacy. How about removing it and start using debugger or unit-tests for inspecting what's in the code?